### PR TITLE
Sphinx Documentation for Presto YARN, Maven to build with prestodb & Teradata Presto

### PR DESCRIPTION
**Pull request by mattsfuller**
https://prestodb.io/presto-yarn/index.html

### CODE REVIEW

**Comment by mattsfuller** [comment]
@anusudarsan can you please review?

File: presto-yarn-docs/src/main/sphinx/deployment/ambari.rst [hunk 1]
```diff
@@ -0,0 +1,68 @@
+Automated Installation Apache Ambari Slider View
+=================================================
```
Comment (anusudarsan) [inline presto-yarn-docs/src/main/sphinx/deployment/ambari.rst]: Installation using

**Comment by anusudarsan** [comment]
Thanks a lot for doing this. Did you copy the content from teradata-docs or presto-yarn/README?

**Comment by mattsfuller** [comment]
AC.
Yes, I copied from presto-yarn/README. 
But I intend to apply changes Brenda made in her PR on top of this.

**Comment by mattsfuller** [comment]
It seems running mvn at the top directory does not invoke the presto-yarn-docs pom.xml.

File: presto-yarn-docs/src/main/sphinx/debugging.rst [hunk 2]
```diff
@@ -0,0 +1,31 @@
+Debugging and Logging
+=====================
+
+Once the YARN application is launched, you can monitor the status at YARN ResourceManager WebUI. 
+
+A successfully launched application will be in ``RUNNING`` state. The YARN ApplicationMaster UI (eg: ``http://master:8088/cluster/app/application_<id>``) will show slider-appmaster, COORDINATOR and WORKER components and the associated containers running based on your configuration. You can also use Slider cli script to check [status](#status).
+
```
Comment (anusudarsan) [inline presto-yarn-docs/src/main/sphinx/debugging.rst]: the link to (#status) has to be updated, right?like -  status <#check-the-status>

**Comment by anusudarsan** [comment]
Are you going to update the links to headers within the docs (see my comment)? They are still using the .md format. Brenda's PR had the links added appropriately

Also you will have to add presto-yarn-docs to the list of module in the `presto-yarn/pom.xml` (line 59) to build it from top project dir.

File: presto-yarn-docs/src/main/sphinx/deployment/ambari.rst [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+::
+   
+   su hdfs
+   hdfs dfs -mkdir -p /user/yarn
+   hdfs dfs -chown yarn:yarn /user/yarn
+
+
+#. Make sure you change the ``global.presto_server_port`` from 8080 to some other unused port, since Ambari by default uses 8080.
+
+#. Make sure the data directory in the UI (added in ``appConfig-default.json`` eg: ``/var/lib/presto/``) is pre-created on all nodes and the directory must be owned by ``global.app_user``, otherwise slider will fail to start Presto with permission errors.
```
Comment (anusudarsan) [inline presto-yarn-docs/src/main/sphinx/deployment/ambari.rst]: @mattsfuller while you are here, can you correct `the directory must be owned by global.app_user`  ->  `the directory must be owned by user 'yarn'`.

**Comment by mattsfuller** [comment]
@kokosing Can you review please?

**Pull request merged**